pkg/controller/jobs/appwrapper/appwrapper_controller.go

```diff
@@ -128,11 +130,28 @@ func (aw *AppWrapper) PodSets() ([]kueue.PodSet, error) {
 	}
 	podSets := make([]kueue.PodSet, len(podSpecTemplates))
 	for psIndex := range podSpecTemplates {
+		var podIndexLabel *string
+		var subGroupIndexLabel *string
+		var subGroupCount *int32
+		if annotation, ok := awPodSets[psIndex].Annotations[awutils.PodSetAnnotationTASPodIndexLabel]; ok {
+			podIndexLabel = &annotation
+		}
+		if annotation, ok := awPodSets[psIndex].Annotations[awutils.PodSetAnnotationTASSubGroupIndexLabel]; ok {
+			subGroupIndexLabel = &annotation
+		}
+		if annotation, ok := awPodSets[psIndex].Annotations[awutils.PodSetAnnotationTASSubGroupCount]; ok {
+			if count, err := strconv.Atoi(annotation); err == nil {
+				subGroupCount = ptr.To[int32](int32(count))
```

Comment:
```suggestion
				subGroupCount = ptr.To(int32(count))
```

This type parameter does not do anything.

Reply:
removed